src/js/view/components/LogViewer.js

```diff
@@ -56,6 +56,11 @@ const LogViewer = props => {
   const lastSeenLogSize = props.lastSeenLogSizes[props.source.path]
     ? props.lastSeenLogSizes[props.source.path]
     : 0;
+  
+  const scrollPositionFromReducer = props.scrollPositions[props.source.path] 
```

Comment:
The scrollPositionFromReducer can probably be renamed to just scrollPosition. When we pairprogrammed the other day I just wanted to call it something different from the variable to distinguish it from the state variable we were replacing.

Reply:
I renamed scrollPositionFromReducer in LogViewer to scrollPosition.